lets-plot-compose/src/androidMain/kotlin/org/jetbrains/letsPlot/skia/compose/android/PlotViewContainer.kt

```diff
@@ -172,5 +203,9 @@ internal class PlotViewContainer(
         }
 
         removeAllViews()
+
+        if (reusePlotSvgPanel) {
+            plotSvgRegistration.dispose()
```

Comment:
Can we do: `svgPanel.dispose()` ?

Reply:
`SvgPanel.dispose()` actually happens few lines above - when we iterate through the children and call `dispose()`. I have rewritten this code as we can now only have one child element.